Notebooks/verify_source_table.ipynb

```diff
@@ -0,0 +1,323 @@
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "all_ids = pd.DataFrame(source_table.get_quantities(['sourceId', 'objectId', 'ra', 'dec', 'mag', 'mag_err']))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "all_ids['mag_err']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bright_ids = all_ids[all_ids['mag_err'] < 0.05]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "len(bright_ids)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.hexbin(bright_ids['mag'], bright_ids['mag_err'])\n",
+    "plt.xlabel('mag')\n",
+    "plt.ylabel('mag_err')\n",
+    "plt.xlim(14, 25)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Looking for objectId for one source\n",
+    "Let's look for all matching Sources for one object.\n",
+    "Let's just take the first entry from the bright_ids, assuming bright sources should be detected in the coadd and have an objectId.\n",
+    "\n",
+    "Basic checks:\n",
+    "  * Are these RA, Dec values agree?\n",
+    "  * Do they agree with the Object Table RA, Dec?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "this_object_id = bright_ids['objectId'].values[0]\n",
+    "these_sources = bright_ids[bright_ids['objectId'] == this_object_id] "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(these_sources[['sourceId', 'objectId', 'ra', 'dec']])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "object_df = pd.DataFrame(object_table.get_quantities(['id', 'ra', 'dec']))\n",
+    "this_object = object_df[object_df['id'] == this_object_id]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "this_object"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Yes!  Looks good."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "source_table.list_all_quantities()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "resid = source_table['mag'] - (-2.5*np.log10(source_table['slot_PsfFlux_instFlux']/source_table['fluxmag0']))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(resid)"
```

Comment:
If this notebook is meant to be a mini tutorial later, maybe it'd be good to visualize the residuals.

Reply:
Yes, I did at first plot them.  But they're supposed to all be exactly 0.  So it was just a line of circles and matplotlib makes up some plot range and it's not clear that they're exactly 0.  I found it easier to print and get the first part and the ending part.

Reply:
But, yes, this Notebook is clearly me convincing myself rather than thoroughly convincing anyone else.